grakn-test/src/test/java/ai/grakn/test/engine/controller/CommitLogControllerTest.java

```diff
@@ -95,6 +95,7 @@ public void resetMockitoMockCounts(){
     }
 
     @Test
+    @Ignore
```

Comment:
We have this one for all the commit log controller tests: https://work.grakn.ai/entity/15095